perfkitbenchmarker/linux_benchmarks/cloud_spanner_ycsb_benchmark.py

```diff
@@ -170,14 +183,20 @@ def Run(benchmark_spec):
     run_kwargs['cloudspanner.host'] = FLAGS.cloud_spanner_host
 
   load_kwargs = run_kwargs.copy()
-  if FLAGS['ycsb_preload_threads'].present:
-    load_kwargs['threads'] = FLAGS.ycsb_preload_threads
+  if not FLAGS['ycsb_preload_threads'].present:
```

Comment:
This follows the pattern in cloud_bigtable_ycsb_benchmark.py, and I think there is a reason for that.

If I have to guess, I think this was not set at some previous version of YCSB. If so, I think we should keep it here, because user can use any YCSB version in the test.